presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -67,6 +67,12 @@ Mathematical Functions
     a real value and the standard deviation must be a real and positive value.
     The probability p must lie on the interval (0, 1).
 
+.. function:: normal_cdf(mean, sd, v) -> double
+
+    Compute the Normal cdf with given mean and standard deviation (sd):  P(N < v; mean, sd).
```

Comment:
I think `cdf` abbrev should be expanded (same in `inverse_normal_cdf`'s docs). 
@electrum would you agree?